site/app/views/submission/HomeworkView.php

```diff
@@ -726,15 +726,25 @@ function submitStudentGradeable(student_id, highest_version) {
 HTML;
 
                                     $title = "";
+                                    $visible1 = "visible";
                                     $return .= <<<HTML
                             <div class='diff-element'>
 HTML;
                                     if ($diff_viewer->hasDisplayExpected()) {
                                         $title = "Student ";
                                     }
+                                    if ($diff_viewer->hasDisplayActual()) {
+                                        //I unfornately have no idea how the strlen is measured. It's not the number of characters in there
+                                        //an extra 250ish characters are added on for some reason.
+                                        if (strlen($diff_viewer->getDisplayActual()) < 350) {
```

Comment:
The characters were a bunch of spaces, html tags and newlines. I made a commit to get rid of them when calculating the strlen.